applications/nrf_desktop/README.rst

```diff
@@ -2154,6 +2155,7 @@ These are valid for events that have many listeners or sources, and are gathered
    doc/qos.rst
    doc/selector.rst
    doc/smp.rst
+   doc/smp_custom.rst
```

Comment:
I am planning to fix it in the next patch update. The name changed and I forgot to align this line